Cargo.toml

```diff
@@ -102,75 +102,75 @@ inquire = { version = "0.7.5" }
-sp-keyring = { default-features = false, git = "https://github.com/input-output-hk/polkadot-sdk.git", tag ="partnerchains-stable-2407-1-v2" }
-sp-keystore = { default-features = false, git = "https://github.com/input-output-hk/polkadot-sdk.git", tag ="partnerchains-stable-2407-1-v2" }
-sp-offchain = { default-features = false, git = "https://github.com/input-output-hk/polkadot-sdk.git", tag ="partnerchains-stable-2407-1-v2" }
-sp-runtime = { default-features = false, git = "https://github.com/input-output-hk/polkadot-sdk.git", tag ="partnerchains-stable-2407-1-v2" }
-sp-session = { default-features = false, git = "https://github.com/input-output-hk/polkadot-sdk.git", tag ="partnerchains-stable-2407-1-v2" }
-sp-tracing = { default-features = false, git = "https://github.com/input-output-hk/polkadot-sdk.git", tag ="partnerchains-stable-2407-1-v2" }
-sp-staking = { default-features = false, git = "https://github.com/input-output-hk/polkadot-sdk.git", tag ="partnerchains-stable-2407-1-v2" }
-sp-std = { default-features = false, git = "https://github.com/input-output-hk/polkadot-sdk.git", tag ="partnerchains-stable-2407-1-v2" }
-sp-timestamp = { default-features = false, git = "https://github.com/input-output-hk/polkadot-sdk.git", tag ="partnerchains-stable-2407-1-v2" }
-sp-transaction-pool = { default-features = false, git = "https://github.com/input-output-hk/polkadot-sdk.git", tag ="partnerchains-stable-2407-1-v2" }
-sp-version = { default-features = false, git = "https://github.com/input-output-hk/polkadot-sdk.git", tag ="partnerchains-stable-2407-1-v2" }
-sp-storage = { default-features = false, git = "https://github.com/input-output-hk/polkadot-sdk.git", tag ="partnerchains-stable-2407-1-v2" }
-sp-weights = { default-features = false, git = "https://github.com/input-output-hk/polkadot-sdk.git", tag ="partnerchains-stable-2407-1-v2" }
-substrate-build-script-utils = { default-features = false, git = "https://github.com/input-output-hk/polkadot-sdk.git", tag ="partnerchains-stable-2407-1-v2" }
-substrate-frame-rpc-system = { default-features = false, git = "https://github.com/input-output-hk/polkadot-sdk.git", tag ="partnerchains-stable-2407-1-v2" }
-substrate-prometheus-endpoint = { package = "substrate-prometheus-endpoint", git = "https://github.com/input-output-hk/polkadot-sdk.git", tag ="partnerchains-stable-2407-1-v2" }
-substrate-test-runtime-client = { default-features = false, git = "https://github.com/input-output-hk/polkadot-sdk.git", tag ="partnerchains-stable-2407-1-v2" }
-substrate-wasm-builder = { default-features = false, git = "https://github.com/input-output-hk/polkadot-sdk.git", tag ="partnerchains-stable-2407-1-v2" }
-sp-genesis-builder = { default-features = false, git = "https://github.com/input-output-hk/polkadot-sdk.git", tag ="partnerchains-stable-2407-1-v2" }
+frame-benchmarking = { default-features = false, git = "https://github.com/input-output-hk/polkadot-sdk.git", tag = "polkadot-stable2407-1-plus-pc-1" }
```

Comment:
This tag has only our changes already merged to master, see diff to stable2407-1:

https://github.com/paritytech/polkadot-sdk/compare/polkadot-stable2407-1...input-output-hk:polkadot-sdk:polkadot-stable2407-1-plus-pc